app/models/user.rb

```diff
@@ -6,4 +6,8 @@ class User < ApplicationRecord
   def recent_posts
     posts.order(title: :desc).limit(3)
   end
```

Comment:
- [ ] Let's test this callback function so that we are sure it will return the 3 most recent posts.